x-pack/plugins/alerting/server/lib/convert_es_sort_to_event_log_sort.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Thanks for the refactor. This function might make more sense inside the event log plugin? WDYT?